server/opts.go

```diff
@@ -4279,16 +4279,16 @@ func parseAllowResponses(v interface{}, errors, warnings *[]error) *ResponsePerm
 }
 
 // Helper function to parse old style authorization configs.
-func parseOldPermissionStyle(v interface{}, errors, warnings *[]error) (*SubjectPermission, error) {
-	subjects, err := parsePermSubjects(v, errors, warnings)
+func parseOldPermissionStyle(v interface{}, errors *[]error) (*SubjectPermission, error) {
```

Comment:
Should we move interface{} -> any in this PR too maybe?

Reply:
Hopefully this does not "break" public APIs? I know `any` is just `interface{}`, but who knows...